mlflow/langchain/__init__.py

```diff
@@ -529,10 +531,12 @@ def load_retriever(persist_directory):
                     )
 
             See a complete example in examples/langchain/retrieval_qa_chain.py.
-        example_no_conversion: If ``False``, the input example will be converted to a Pandas
+        example_no_conversion: This parameter is deprecated and will be removed in a future
+                release, please drop it and input examples are not converted anymore.
```

Comment:
Just to confirm, `example_no_conversion` is no longer used at all, correct?

Reply:
Yes